dist/sinon-chrome.latest.js

```diff
@@ -54,11 +54,6 @@ return /******/ (function(modules) { // webpackBootstrap
 /* 0 */
```

Comment:
find out some way to ignore `dist/` in git otherwise it would be difficult to review prs.
I suggest to setup nightly builds and put artifacts somewhere (e.g. cdn or github pages)